### DISCUSSION TITLE
Weird colour in VIM.

### DISCUSSION THREAD

**Discussion by anurag3301** [kovidgoyal/kitty#3767] [Q&A] [answered]
Why I'm getting such weird colour randomly around text in vim.
[![2021-06-25-230043-1364x767-scrot.png](https://i.postimg.cc/bYMX93b8/2021-06-25-230043-1364x767-scrot.png)](https://postimg.cc/mhQ6b386)
The exact same screenshot from terminator. Is it a kitty issue or vim issue?
[![2021-06-25-230157-1367x767-scrot.png](https://i.postimg.cc/1X6ZqqvD/2021-06-25-230157-1367x767-scrot.png)](https://postimg.cc/n9H58CbL)

**Comment by Luflosi** [comment, marked as answer]
https://sw.kovidgoyal.net/kitty/faq.html#using-a-color-theme-with-a-background-color-does-not-work-well-in-vim

**Reply by anurag3301** [reply]
Thanks buddy, I worked!!  😊
